src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx

```diff
@@ -62,15 +69,24 @@ const YieldBearingRewards = observer(({ open, onModalClose }: YieldBearingReward
       aria-describedby="alert-dialog-description"
     >
       <DialogTitle id="alert-dialog-title" className={classes.title}>
-        <Typography variant="h6" color="primary" className={classes.titleText}>
-          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" /> Yield-Bearing Rewards
-        </Typography>
+        <Box className={classes.titleText}>
+          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" />
+          <Typography variant="h6" color="primary">
+            Yield-Bearing Rewards
+          </Typography>
+        </Box>
         <IconButton aria-label="close" className={classes.closeButton} onClick={onModalClose}>
           <CloseIcon />
         </IconButton>
       </DialogTitle>
       <DialogContent id="alert-dialog-description" className={classes.root}>
-        <Box className={classes.video}></Box>
+        {/* <Box className={classes.video}>
```

Comment:
do we want this commented out?

Reply:
yeah will wants no embed till he makes the youtube video which is going to be latter than the release date for this